pkgs/tools/inputmethods/ibus/default.nix

```diff
@@ -34,6 +34,7 @@
 , buildPackages
 , runtimeShell
 , nixosTests
+, ibus
```

Comment:
Is it needed? @jtojnar

Reply:
Thanks, that was a remnant of previous attempt.